Flexible Environments - clarify link to Merge Conflicts article (#7205)
umbraco/UmbracoDocs: umbraco-cloud/troubleshooting/deployments/README.md

* [SQL Timeouts](https://docs.umbraco.com/umbraco-deploy/deploy-settings#timeout-settings).
* [Dependency Exception](dependency-exceptions.md).
* [Media path too long](path-too-long-exception.md).

> With the introduction of Flexible Environments the change of getting merge conflicts on Umbraco Cloud have increased. Read more about what to do in case of Merge conclicts and how to avoid them. 

Reviewer: ```suggestion
With the introduction of Flexible Environments, the chance of getting merge conflicts on Umbraco Cloud has increased. Read more about what to do in case of merge conflicts and how to avoid them. 
```
